packages/edit-site/src/components/left-sidebar/navigation-panel/constants.js

```diff
@@ -0,0 +1,68 @@
+export const ITEM_CONTENTS = {
```

Comment:
I think we could clarify these constants a little bit.

Something like:
- `ITEM_CONTENTS` -> `TEMPLATES_DEFAULT_DETAILS`
- `GENERAL_TEMPLATES` -> `TEMPLATES_GENERAL` (nitpick)
- `POSTS_GENERAL_TEMPLATES` -> `TEMPLATES_POSTS` (nitpick, but we could lose the "general" here 🤔 )